lib/CheckStatusHook.ts

```diff
@@ -3,14 +3,13 @@ import { ApiError } from 'next/dist/server/api-utils'
 import {
```

Comment:
Filename should be `useCheckStatus`.

Reply:
File names don't have hard fast rules, adding the key word "use" provides no context as to what this file is for. This updated name make that far more clear.

Reply:
`useSomething` is well known in the react to be a hook. Component naming convention is PascalCase. Functions, variables and props should be in camelCase. TS types, interfaces and enums should be in PascalCase. I suggest to follow what's is being used within react community.

example from MUI for hook naming:
https://github.com/mui/material-ui/tree/master/packages/mui-material/src/useMediaQuery

Reply:
The file isn't just that function; It's all the functions related to the hook.
My personal preference would also have me change the name, but I am leaving it.

Reply:
It's either that we move the fetcher function in his own file or we get ridd of it and move the method logic inside useQuery `queryFn` parameter.

Reply:
I don't see the need of that in this moment and would be out of scope for this PR.